src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildAssetsTest.cs

```diff
@@ -150,16 +150,17 @@ public void FullPath ()
 			};
 			using (var b = CreateDllBuilder (Path.Combine ("temp", TestName, "SubDir"))) {
 				Assert.IsTrue (b.Build (proj), "Build should have succeeded.");
-				var libraryProjectImports = Path.Combine (Root, b.ProjectDirectory, proj.IntermediateOutputPath, "__AndroidLibraryProjects__.zip");
-				FileAssert.Exists (libraryProjectImports);
-				using (var zip = ZipHelper.OpenZip (libraryProjectImports)) {
-					var entryName = "library_project_imports/assets/foo.txt";
-					var entry = zip.ReadEntry (entryName);
-					Assert.IsNotNull (entry, $"{libraryProjectImports} should contain {entryName}");
-					using (var memory = new MemoryStream ()) {
-						entry.Extract (memory);
-						memory.Position = 0;
-						Assert.AreEqual ("bar", Encoding.Default.GetString (memory.ToArray ()));
+				if (Builder.UseDotNet) {
+					var aarPath = Path.Combine (Root, b.ProjectDirectory, proj.OutputPath, $"{proj.ProjectName}.aar");
+					FileAssert.Exists (aarPath);
```

Comment:
just a thought. do we need a test to make sure that we DONT include any EmbeddedResource like `__AndroidLibraryProjects__.zip` for dotnet projects?

Reply:
I added `FileAssert.DoesNotExist (libraryProjectImports);`, and in another test, did this:

```csharp
// Check EmbeddedResource files do not exist
using (var assembly = AssemblyDefinition.ReadAssembly (assemblyPath)) {
	Assert.AreEqual (0, assembly.MainModule.Resources.Count);
}
```